### PR TITLE
fix some issues with 1.127 & cherry-pick cower emote from LosOjos

### PR DESCRIPTION
**Pull request by JVirant**

### CODE REVIEW

File: GameServer/serverrules/AbstractServerRules.cs [hunk 1]
```diff
@@ -2129,22 +2129,27 @@ public virtual void SendHousingMerchantWindow(GamePlayer player, DOL.GS.PacketHa
...
 				case eMerchantWindowType.HousingNPCHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.GetNpcShopItems(player), eMerchantWindowType.HousingNPCHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.GetNpcShopItems(player), merchantType);
 					break;
 				case eMerchantWindowType.HousingVaultHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorVaultShopItems, eMerchantWindowType.HousingVaultHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorVaultShopItems, merchantType);
+					break;
+				case eMerchantWindowType.HousingDeedMenu:
+					player.Out.SendMerchantWindow(/* TODO */HouseTemplateMgr.OutdoorMenuItems, eMerchantWindowType.HousingDeedMenu);
```
Comment (JVirant) [inline GameServer/serverrules/AbstractServerRules.cs:2152]: I don't have any idea about what this list should contains... In this PR, I was focused to make 1.127 works.

**Comment by NetDwarf** [comment]
What are the "some issues" fixed?

**Comment by JVirant** [comment]
> 
> 
> What are the "some issues" fixed?

It's explained in the commit message

**Comment by NetDwarf** [comment]
> > What are the "some issues" fixed?
> 
> It's explained in the commit message

Sorry, missed it :sweat_smile:

**Comment by NetDwarf** [comment]
Thank you! :+1:

**Pull request merged**